src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/form/common/product/ProductGridContext.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+import {useTranslate} from '@akeneo-pim-community/shared';
+
+type Props = {
+  type: string;
+  children: string;
+};
+
+const ProductGridContext = ({type, children}: Props) => {
+  const translate = useTranslate();
+
+  return (
+    <ContextContainer>
+      <ViewNameContainer>{children}</ViewNameContainer>{' '}
+      {type === 'public' || type === 'view' ? `(${translate('pim_common.public_view')})` : null}
```

Comment:
In order to ease the readability of the code, we could manage the separating space directly in the template string like below:

```suggestion
      <ViewNameContainer>{children}</ViewNameContainer>
      {type === 'public' || type === 'view' ? ` (${translate('pim_common.public_view')})` : null}
```